libs/coin-evm/docs/getTransactionStatus.md

```diff
@@ -0,0 +1,25 @@
+# `getTransactionStatus.ts`
```

Comment:
duplicated doc (this one is more consistent than the one we currently have)

validateEditTransaction is already commented in code but missing here, that's why i think we should move everything into the code

Reply:
Like commented before, I'd advise to keep the 2 types of documentation separated.

`validateEditTransaction` has been added by Alexandre after I made this doc, which explains why it's not defined here. I'll add it 👍